asm/trafficdirector/td-configmap-galley.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-galley-config-td
+  namespace: istio-system
+data:
+  TRAFFICDIRECTORADDRESS: "trafficdirector.googleapis.com:443" # {"$ref":"#/definitions/io.k8s.cli.substitutions.trafficdirector-address"}
+  MESHCAADDRESS: "meshca.googleapis.com:443" # {"$ref":"#/definitions/io.k8s.cli.substitutions.meshca-address"}
```

Comment:
I guess you can directly use setters here: `# {"$ref":"#/definitions/io.k8s.cli.anthos.servicemesh.meshca-address"}`